action.yml

```diff
@@ -360,4 +360,4 @@ runs:
       if: ${{ always() && inputs.disableTelemetry == 'false' }}
       shell: bash
       run: |
-        docker run --rm mcr.microsoft.com/oryx/build:github-actions-debian-buster-20230206.2  /bin/bash -c "oryx telemetry --event-name 'ContainerAppsGitHubAction' ${{ env.CA_GH_ACTION_LENGTH_MILLISECONDS_ARG }} ${{ env.CA_GH_ACTION_SCENARIO_ARG }} ${{ env.CA_GH_ACTION_RESULT_ARG }}"
\ No newline at end of file
+        docker run --rm mcr.microsoft.com/oryx/cli:debian-buster-20230207.2  /bin/bash -c "oryx telemetry --event-name 'ContainerAppsGitHubAction' ${{ env.CA_GH_ACTION_LENGTH_MILLISECONDS_ARG }} ${{ env.CA_GH_ACTION_SCENARIO_ARG }} ${{ env.CA_GH_ACTION_RESULT_ARG }}"
```

Comment:
Out of curiosity, could you please clarify why not using the same image everywhere (for example `mcr.microsoft.com/oryx/cli:builder-debian-buster-20230208.1`)? I assumed it will get the same `oryx telemetry` capability, but maybe there is a subtlety? Or is it because this image is smaller?

Reply:
The CLI image we use for the builder is bigger because we pre-bake more into it since we cannot currently defer installing the platform-specific `apt-get` calls (and other build-time dependencies) when we generate the build script, so it makes it tricky to justify using that since it's not guaranteed the user is targeting the builder and will already have pulled this image down. I'm using the latest CLI image that Oryx publishes since it _does_ rely on this logic, which makes the image significantly smaller, and since we only need the image to call `oryx telemetry`, it makes the pulling and command call much quicker.

Reply:
Thanks for clarifying!